pages/take-action.vue

```diff
@@ -54,7 +54,16 @@
                     </v-btn>
                 </div>
             </v-row>
-            <v-row justify="center">
+            <v-row justify="center" align="center" class="my-2">
+                <div class="Take-Action-Title"> Take Action </div>
+            </v-row>
+            <v-row justify="center" align="center" class="mb-10">
+                <div class="Take-Action-Subheading"> See something you don't like? Use your voice to create change! </div>
+            </v-row>
+            <!-- <v-row class="my-5"> <v-spacer /> <v-col col="1">
+                <v-text-field label="Enter Your Zip Code" outlined placeholder="94110" light dense background-color="white"></v-text-field>
+            </v-col> <v-spacer /> </v-row> -->
```

Comment:
might as well just remove, we can find it from the git history later if we want to add back in
```suggestion
```

Reply:
cool makes sense for keeping everything cleaner